dali/imgcodec/decoders/jpeg/jpeg_handle.cc

```diff
@@ -36,6 +36,48 @@ void CatchError(j_common_ptr cinfo) {
   longjmp(*jpeg_jmpbuf, 1);
 }
 
+inline int GetLibJpegMaxProgressiveScansEnv() {
+  static constexpr int kDefaultScansNum = 500;
+  char *env = getenv("DALI_MAX_JPEG_SCANS");
```

Comment:
that's a good start. I would consider adding an operator parameter, but it can be added later.

Reply:
I though about exposing it as an operator parameter, but in this particular case, I believe higher values (even this one) are unlikely. Reportedly, typical values are up to 10.

Reply:
And I took 500 from libjpeg-turbo TJ API - there, if you specify a boolean flag to limit the scans they pick 500. Libtiff, on the other hand, goes for 100. In any case, I don't expect users to really adjust that.

Reply:
256 sounds like a good candidate then.